### PR TITLE
Exporter start script changes

### LINKED ISSUE #2244: Missing PG_STAT_STATEMENTS_LIMIT
```
time="2021-01-30T10:24:24Z" level=info msg="Error running query on database \"127.0.0.1:5432\": ccp_pg_stat_statements_top_total pq: column \"pg_stat_statements_limit\" does not exist" source="postgres_exporter.go:1356"
time="2021-01-30T10:24:24Z" level=info msg="Error running query on database \"127.0.0.1:5432\": ccp_pg_stat_statements_top_mean pq: column \"pg_stat_statements_limit\" does not exist" source="postgres_exporter.go:1356"
time="2021-01-30T10:24:24Z" level=info msg="Error running query on database \"127.0.0.1:5432\": ccp_pg_stat_statements_top_max pq: column \"pg_stat_statements_limit\" does not exist" source="postgres_exporter.go:1356"
time="2021-01-30T10:24:24Z" level=error msg="queryNamespaceMappings returned 3 errors" source="postgres_exporter.go:1474"
```

A new env variable was introduced in: https://github.com/CrunchyData/pgmonitor/blob/master/exporter/postgres/queries_pg_stat_statements_pg13.yml

https://github.com/CrunchyData/postgres-operator/blob/775fd1e8450fab5289e72a79617cc693f7f72c51/bin/crunchy-postgres-exporter/start.sh#L217
This line is incomplete.

### PR DESCRIPTION
**Pull request by ssiahetiong**
**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
Fix #2244 

**What is the new behavior (if this is a feature change)?**
1. Modify pgoption to align with https://github.com/CrunchyData/pgmonitor/blob/853afdfc7d7b59a38e4fd20da7bdc058217be5ef/exporter/postgres/sysconfig.postgres_exporter_pg13#L10
2. Added default value for PG_STAT_STATEMENTS_LIMIT to prevent errors for pg version 13 

**Other information**:

### CODE REVIEW

File: bin/crunchy-postgres-exporter/start.sh [hunk 1]
```diff
@@ -214,9 +214,9 @@ else
     fi
 fi
 
-sed -i "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES:-10}/g" /tmp/queries.yml
+sed -i -e "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES:-10}/g" -e "s/#PG_STAT_STATEMENTS_LIMIT#/${PG_STAT_STATEMENTS_LIMIT:-100}/g" /tmp/queries.yml
 
-PG_OPTIONS="--extend.query-path=${QUERY_DIR?}/queries.yml  --web.listen-address=:${POSTGRES_EXPORTER_PORT}"
+PG_OPTIONS="--extend.query-path=${QUERY_DIR?}/queries.yml  --web.listen-address=:${POSTGRES_EXPORTER_PORT} --disable-default-metrics --disable-settings-metrics"
```
Comment (jkatz) [inline bin/crunchy-postgres-exporter/start.sh:219]: Why is this being changed?

**Review by jkatz — CHANGES_REQUESTED** [review]
I will work on testing this today. I left a comment on one of the changes.

Please write commit messages that align with the contribution guidelines:

[https://github.com/CrunchyData/postgres-operator/blob/master/CONTRIBUTING.md#commit-messages](https://github.com/CrunchyData/postgres-operator/blob/master/CONTRIBUTING.md#commit-messages)

These guidelines are in place to help others understand why the changes are being made.

Thanks for the contribution!

File: bin/crunchy-postgres-exporter/start.sh (hunk 1 above)
Comment (ssiahetiong) [reply bin/crunchy-postgres-exporter/start.sh:219]: The additional parameters were present on the pgmonitor sysconfig file:
https://github.com/CrunchyData/pgmonitor/blob/853afdfc7d7b59a38e4fd20da7bdc058217be5ef/exporter/postgres/sysconfig.postgres_exporter_pg13#L10

I modified it to re-align with the pgmonitor standalone setup.

Comment (jkatz) [reply bin/crunchy-postgres-exporter/start.sh:219]: @ssiahetiong Unfortunately the `--disable-settings-metrics` breaks the existing integration. For example, when testing, I found that the active connection dials no longer worked.

While the Postgres Operator uses a lot of the functionality brought by pgMonitor, it does rely on some of the `postgres_exporter` defaults to accomplish it tasks.

Please rebase and remove 0f118f94bdca73f460c8960e2bb83f04e0398e21 from this patchset. Thanks!

**Comment by jkatz** [comment]
710caf2 appears to work for me. Doing one more test on my local without 0f118f9.

File: bin/crunchy-postgres-exporter/start.sh (hunk 1 above)
Comment (ssiahetiong) [reply bin/crunchy-postgres-exporter/start.sh]: done

File: bin/crunchy-postgres-exporter/start.sh [hunk 2]
```diff
@@ -214,7 +214,7 @@ else
     fi
 fi
 
-sed -i "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES:-10}/g" /tmp/queries.yml
+sed -i -e "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES:-10}/g" -e "s/#PG_STAT_STATEMENTS_LIMIT#/${PG_STAT_STATEMENTS_LIMIT:-100}/g" /tmp/queries.yml
```
Comment (jkatz) [inline bin/crunchy-postgres-exporter/start.sh:217]:
```suggestion
sed -i -e "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES:-10}/g" -e "s/#PG_STAT_STATEMENTS_LIMIT#/${PG_STAT_STATEMENTS_LIMIT:-20}/g" /tmp/queries.yml
```

Modeling the upstream pgMonitor change for `PG_STAT_STATEMENTS_LIMIT`:

https://github.com/CrunchyData/pgmonitor/blob/41ef7002498c9635880b0054642fd9dd300cc66c/exporter/postgres/sysconfig.postgres_exporter_pg13#L9

**Review by jkatz — CHANGES_REQUESTED** [review]
One more change and then this is good to go!

**Review by jkatz — APPROVED** [review]

**Pull request merged**